addons/hr_holidays/views/hr_holidays_views.xml

```diff
@@ -675,8 +697,7 @@
             <field name="type">ir.actions.act_window</field>
             <field name="res_model">hr.holidays.status</field>
             <field name="view_type">form</field>
-            <field name="view_mode">tree,form</field>
-            <field name="view_id" eval="view_holiday_status_normal_tree"/>
```

Comment:
We don't want to use view_holiday_status_normal_tree anymore?

Reply:
view_holiday_status_normal_tree is needed.